### PR TITLE
Disable cookie config saving when using resource calendar

### PR DESCRIPTION
**Pull request by Guntrisoft**
- Store event changes with XHR
- Ensures that resource calendar always defaults to today
  when returning back to the calendar
- Ensures that the company calendar is unaffected by this
  change

### CODE REVIEW

File: spec/features/resource_manager_modifies_appointments_spec.rb [hunk 1]
```diff
@@ -138,7 +137,7 @@ def and_there_is_a_bookable_slot_for_a_guider
     )
   end
 
-  def then_they_can_see_the_bookable_slot
+  def then_they_can_see_the_bookable_slot # rubocop:disable Metrics/AbcSize
```
Comment (benlovell) [inline spec/features/resource_manager_modifies_appointments_spec.rb]: Just disable it globally at the top of the file instead of repeating everywhere, especially since actually disabling it once in the file disables the rule for the rest of the file from that line.

Comment (benlovell) [inline spec/features/resource_manager_modifies_appointments_spec.rb]: TIL but still pointless. It's a spec file so we apply different heuristics to production code. We should just globally disable this rule in the rubocop configuration:

```
# .rubocop.yml

Metrics/AbcSize:
  Exclude:
    - 'spec/features/*'
```

Comment (AndrewVos) [inline spec/features/resource_manager_modifies_appointments_spec.rb]: @benlovell Fair enough. I would do that as another PR though, because of all the deletions that will have to happen.
